day-4--part2/hasMatchingPair.js

```diff
@@ -1,4 +1,18 @@
-const isSimilar = (a, b) => parseInt(a) === parseInt(b);
+const isEqual = (a, b) => parseInt(a) === parseInt(b);
+
+/**
+ * Check around pair to make sure its only a pair ("11")
+ * and not multiple same digits ("111")
+ *
+ * >b should not equal a and c should not equal a
+ *
+ * @param {String} a
+ * @param {String} b
+ * @param {String} c
+ */
+const isStrictlyPair = (a, b, c) =>
```

Comment:
I think this is better! And the comment above helps a lot. On the subject of code clarity, I would encourage you to think about the parameter names as well. I don't think this matters quite as much for this 20 line file, but in a larger system with more complexity things become harder to follow.

for instance
```js
const isStrictlyPair = (pairValue, beforePair, afterPair) =>
  parseInt(pairValue) !== parseInt(beforePair) &&
  parseInt(pairValue) !== parseInt(afterPair);
```

Something like that ☝️ is helpful when reading the function definition. You can take it a step further and use a single object parameter which can reveal more to the reader in _both_ places (where the function is declared and where it is called) - the tradeoff being verbosity.

Reply:
Ahh! That looks a ton more readable 😄